code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -229,10 +229,6 @@
 	new_character.dna.ready_dna(new_character)
 	new_character.key = G_found.key
 
-	// Create a brand new mind for the dude
-	new_character.mind = new
-	new_character.mind.current = new_character
-
```

Comment:
This caused mobs created with makeBody() to spawn with an uninitialized mind, leading to several more issues such as #33027